models/artifact_helpers.go

```diff
@@ -205,14 +205,14 @@ func CreateArtifact(q *reform.Querier, params CreateArtifactParams) (*Artifact,
 	return row, nil
 }
 
-// ChangeArtifactParams are params for changing existing artifact.
-type ChangeArtifactParams struct {
+// UpdateArtifactParams are params for changing existing artifact.
+type UpdateArtifactParams struct {
```

Comment:
What is the reason to change naming? We use `Change` and `Remove` naming everywhere.

Reply:
> We Remove things we Add, and Delete things we Create.
> We Add pre-existing things (you can’t create email server, for example, you just add its configuration to PMM) and Create new things (we control everything about alerting rule, for example, so we create it)

That's how naming logic was described by AlekSi

Reply:
TBH that sounds very interesting but, looks like it's more related to API level, not model level.

Reply:
I think if we will apply different naming rules to API and models we will be completely confused 😄 

For me, it's ok to have any of these names.